command.go

```diff
@@ -783,3 +783,92 @@ func (cmd *ClusterSlotCmd) parseReply(cn *conn) error {
 	cmd.val = v.([]ClusterSlotInfo)
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+// Location type for GEO operations in Redis
+type GeoLocation struct {
+	Longitude, Latitude, Name, Distance string
```

Comment:
I still don't fully understand why we can't make `Longitude, Latitude, Distance float64`. I think your point is that `Distance = 0` when `WithDist = false` is confusing, which is probably true, but I think it is a minor point and having correct types is more important.

Also what about `Longitude, Latitude`?